extensions/chocolatey-core.extension/extensions/Get-WebContent.psm1

```diff
@@ -0,0 +1,34 @@
+
+function GetPackageCacheLocation () {
+  $chocoTemp      = $env:TEMP
+  $packageName    = $env:chocolateyPackageName
+  $packageVersion = $env:chocolateyPackageVersion
+
+  $packageTemp = Join-Path $chocoTemp $packageName
+  if ($packageVersion) {
+    $packageTemp = Join-Path $packageTemp $packageVersion
+  }
+
+  $packageTemp
+}
+
+function GetTempFileName () {
+  $tempDir = GetPackageCacheLocation
+  $fileName = [System.IO.Path]::GetRandomFileName()
+
+  $tempFile = Join-Path $tempDir $fileName
+
+  $tempFile
+}
+
+function Get-WebContent ([string]$url) {
+  $filePath = GetTempFileName
+  Get-WebFile -url $url -fileName $filePath

```

Comment:
@ebugusey have a look at this example:

https://github.com/majkinetor/au-packages/blob/master/nirlauncher/tools/chocolateyInstall.ps1#L9-L10

Sometimes, it is necessary to alter the options that are passed into Get-WebFile, and it would be good if we could continue to set these when calling Get-WebContent.

You are see how these parameters are available here:

https://github.com/chocolatey/choco/blob/c798ae3a1b79101814599143a0ed416a69dc13ff/src/chocolatey.resources/helpers/functions/Get-WebFile.ps1#L85-L91